io/cmd_test.go

```diff
@@ -0,0 +1,97 @@
+package io
+
+import (
+	"errors"
+	"regexp"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var matchAllRegexp = regexp.MustCompile(".*")
+var errParsing = errors.New("parsing error")
+
+func TestRunCmdWithOutputParser(t *testing.T) {
+	config := NewCommand("go", "", []string{"version"})
+	stdout, stderr, exitOk, err := RunCmdWithOutputParser(config, false, &CmdOutputPattern{
+		RegExp:   matchAllRegexp,
+		ExecFunc: func(pattern *CmdOutputPattern) (string, error) { return pattern.Line, nil },
+	})
+	assert.NoError(t, err)
```

Comment:
```suggestion
	parserCalled := false
	stdout, stderr, exitOk, err := RunCmdWithOutputParser(config, false, &CmdOutputPattern{
		RegExp:   matchAllRegexp,
		ExecFunc: func(pattern *CmdOutputPattern) (string, error) {
			parserCalled = true
			return pattern.Line, nil 
		},
	})
	assert.NoError(t, err)
	assert.True(t, parserCalled)
```
lets also make sure the parser is also called at this process